src/coord_finder.cpp

```diff
@@ -10,23 +11,28 @@
 #include <iostream>
 #include <ros/console.h>
 
+/**
+* @brief Finds the relative local coordinates of the prop with the robot's current position as a reference
+* 
+* Using the angle range and LiDAR scan, picks the closest point within the angle range.
+* The props are small enough that this point can be taken to represent the center of the prop. 
+* Converts this distance and angle to x and y coordinates
+*
+*/
 class CoordFinder {
 public:
     CoordFinder() : nh_(""), private_nh_("~") {
```

Comment:
add a comment that you need o figure out why private nh works.

Reply:
I made a task in asana for this instead, this isn't specific to this class.

Reply:
Sounds good to me!